microprot/scripts/process_fasta.py

```diff
@@ -1,21 +1,22 @@
 import os
 import urllib
 import click
+import skbio
```

Comment:
I've heard that importing can be quite expensive. Thus, we should try to be most specific about what to import. Therefore "from skbio import io" seems to be better than "import skbio" - to me.